FEXCore/Source/Interface/Core/Interpreter/Fallbacks/InterpreterFallbacks.cpp

```diff
@@ -88,11 +124,11 @@ bool InterpreterOps::GetFallbackHandler(bool SupportsPreserveAllABI, const IR::I
 
     switch (Op->SrcSize) {
     case IR::OpSize::i32Bit: {
-      *Info = {FABI_F80_I16_F32_PTR, (void*)&FEXCore::CPU::OpHandlers<IR::OP_F80CVTTO>::handle4, Core::OPINDEX_F80CVTTO_4, SupportsPreserveAllABI};
+      *Info = {FABI_F80_I16_F32_PTR, Core::OPINDEX_F80CVTTO_4, SupportsPreserveAllABI};
```

Comment:
A shared Core::OPINDEX to FABI map would be nice since this is effectively duplicated above for indexing ABIHandlers

Reply:
That would require quite a bit more refactoring that I'll think about doing after this PR at some point.